### DISCUSSION TITLE
Is this project open source?

### DISCUSSION THREAD

**Discussion by shadikhan** [oms-tech/reviews#64] [Q&A]
Are we able to clone the repo and create a PR for you to review and merge? Should we submit a feature-request first so that you (Matt or other owners) can approve the development of the feature?

**Comment by loganwishartcraig** [comment]
Also interested in an answer here - I've tried cloning the repo, but the sanity tokens aren't available.

Wanted to try implementing a feature or two, but am at a standstill without a way to get development data locally :(

**Reply by tkwhen** [reply]
@m4ttsch Could you add a seed script that populates sample data?
